### PR TITLE
[ONNX] Extend ONNX Frontend with Function GreaterOrEqual

### LINKED ISSUE #20545: [Good First Issue]: Extend ONNX Frontend with Function GreaterOrEqual
### Context

Neural networks are graphs consisting of nodes called operators. Each operator corresponds to a mathematical function, usually described in framework's documentation or an AI standard, such as ONNX.
OpenVINO ONNX Frontend is a component responsible for working with ONNX graphs and requires implementation of different ONNX operators in order to use ONNX models.
This task requires extending OpenVINO ONNX Frontend with Function GreaterOrEqual.													
Necessary help will be provided by ONNX Fronted team.

### What needs to be done?

Operator details can be found in [ONNX Operators](https://github.com/onnx/onnx/blob/main/docs/Operators.md#GreaterOrEqual)
More details can be found in More details can be found in [ONNX Changelog GreaterOrEqual-12](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#GreaterOrEqual-12)
and [ONNX Changelog GreaterOrEqual-16](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#GreaterOrEqual-16)
1. Create `.hpp` and `.cpp` files for *Windows [here](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/frontend/src/op)
2. Prepare an implementation of this operator in form of a function. GreaterOrEqual-12 should be placed in [opset 1 namespace](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/unique.hpp#L13). GreaterOrEqual-16 should be placed in [opset 16 namespace](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/softmax.cpp#L50).
3. Register the function in [ops_bridge.cpp](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/ops_bridge.cpp#L554-L560) while keeping alphabetical order
4. Create test model(s) in [ONNX models directory](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/tests/models/). OpenVINO test infrastructure then converts prototxt files to ONNX models - you will use those models later in tests
5. Add tests covering all use cases [here](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/tests/onnx_import.in.cpp)
6. Check [Python xfailed tests](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/tests/tests_python/test_backend.py) to find a test marked as a xfailed for added functionality. If any exist - remove corresponding lines and try to verify by using cmdline "python -m pytest -k name_of_test".
More details in [adding operators to ONNX Frontend guide](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/docs/how_to_add_op.md)

### Example Pull Requests

- https://github.com/openvinotoolkit/openvino/pull/15076
- https://github.com/openvinotoolkit/openvino/pull/14936
- https://github.com/openvinotoolkit/openvino/pull/12510
- https://github.com/openvinotoolkit/openvino/pull/19428

### Resources

- [What is OpenVINO?](https://github.com/openvinotoolkit/openvino#what-is-openvino-toolkit)
- [Contribution guide](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
- [Blog post on contributing to OpenVINO](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
- [User documentation](https://docs.openvino.ai/)
- Guides on [ONNX Frontend architecture](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/docs/architecture.md) and [ONNX Frontend tests](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/docs/tests.md) are also available
- You can also see a [blog post on contributing to OpenVINO](https://medium.com/openvino-toolkit/how-to-contribute-to-an-ai-open-source-project-c741f48e009e)

### Contact points

@gkrivor

### Ticket

_No response_

### PR DESCRIPTION
**Pull request by siddhant-0707**
### Details:
 - created implementation for GreaterOrEqual-12, GreaterOrEqual-16  operator
 - created basic tests for them

### Tickets:
 - Closes #20545

### CODE REVIEW

File: src/frontends/onnx/frontend/src/op/greater_or_equal.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    const auto A = node.get_ng_inputs().at(0);
+    const auto B = node.get_ng_inputs().at(1);
+
+    const auto C = std::make_shared<default_opset::GreaterEqual>(A, B);
+
+    return {C};
+}
+}  // namespace set_1
+
+namespace set_12 {
```
Comment (gkrivor) [inline src/frontends/onnx/frontend/src/op/greater_or_equal.cpp:27]: As I can see in docs - it has two versions 12 and 16. In such case initial version 12 should be in set_1, and newer version 16 in set_16. Could you align, please?

File: src/frontends/onnx/frontend/src/op/greater_or_equal.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+OutputVector greater_or_equal(const Node& node) {
+    const auto A = node.get_ng_inputs().at(0);
```
Comment (gkrivor) [inline src/frontends/onnx/frontend/src/op/greater_or_equal.cpp:29]: If we talk about difference between versions - I found only in supported type: version from opset 12 doesn't support bfloat16. Correct me if I'm wrong. So, I suggest to add in set_12 a check:
```cpp
    NGRAPH_CHECK(A.get_element_type() != ov::element::bf16 && B.get_element_type() != ov::element::bf16,
                 "The input data bfloat16 isn't supported in opset 12");
```

File: src/frontends/onnx/frontend/src/ops_bridge.cpp [hunk 3]
```diff
@@ -395,6 +396,8 @@ OperatorsBridge::OperatorsBridge() {
     REGISTER_OPERATOR("GlobalLpPool", 1, global_lp_pool);
     REGISTER_OPERATOR("GlobalMaxPool", 1, global_max_pool);
     REGISTER_OPERATOR("Greater", 1, greater);
+    REGISTER_OPERATOR("Greater_Or_Equal", 1, greater_or_equal);
```
Comment (gkrivor) [inline src/frontends/onnx/frontend/src/ops_bridge.cpp:399]: Also, should be 1 (which is actually oldest, 12) and 16

**Review by gkrivor — CHANGES_REQUESTED** [review]
Hi, good job! Could you fix, please, a code style and some comments about opset 12/16

**Comment by gkrivor** [comment]
build_jenkins

**Review by gkrivor — APPROVED** [review]
